command/pr.go

```diff
@@ -382,10 +382,13 @@ func prSelectorForCurrentBranch(ctx context.Context) (prNumber int, prHeadRef st
 	return
 }
 
-func printPrs(w io.Writer, totalCount int, prs ...api.PullRequest) {
+func printPrs(w io.Writer, totalCount int, printStatus bool, prs ...api.PullRequest) {
 	for _, pr := range prs {
 		prNumber := fmt.Sprintf("#%d", pr.Number)
 		fmt.Fprintf(w, "  %s  %s %s", utils.Green(prNumber), text.Truncate(50, replaceExcessiveWhitespace(pr.Title)), utils.Cyan("["+pr.HeadLabel()+"]"))
+		if printStatus {
+			fmt.Fprintf(w, " (%s)", pr.State)
+		}
```

Comment:
suggestion: How about change the text color based on PR state? 🤔 

```suggestion
		if printStatus {
			if pr.State == "OPEN" {
				fmt.Fprintf(w, " %s", utils.Green("(OPEN)"))
			} else if pr.State == "MERGED" {
				fmt.Fprintf(w, " %s", utils.Magenta("(MERGED)"))
			} else if pr.State == "CLOSED" {
				fmt.Fprintf(w, " %s", utils.Red("(CLOSED)"))
			}
		}
```

Reply:
@ampinsk 
This is how the suggested PR status badge format and color. Just a prompt idea. I don't stick to it at all. 😄 

![Screen Shot 2020-02-25 at 1 19 53](https://user-images.githubusercontent.com/5877477/75171451-fd8b3800-576e-11ea-9695-1ee496a17ee9.png)

A bit OT just for sharing: If the background color is given by "[Solarized dark](https://ethanschoonover.com/solarized/)", `You have no pull requests to review` is not visible at all. 🙈 

![Screen Shot 2020-02-25 at 1 20 17](https://user-images.githubusercontent.com/5877477/75171785-87d39c00-576f-11ea-942c-f07a0bf7f5df.png)